src/chunkserver/network_worker_thread.cc

```diff
@@ -1624,66 +1679,68 @@ void NetworkWorkerThread::preparePollFds() {
 
 	std::unique_lock lock(csservheadLock);
 	for (auto& entry : csservEntries) {
-		entry.pdescpos = -1;
-		entry.fwdpdescpos = -1;
+		entry.pDescPos = -1;
+		entry.fwdPDescPos = -1;
 		switch (entry.state) {
-			case IDLE:
-			case READ:
-			case GET_BLOCK:
-			case WRITELAST:
+			case ChunkserverEntry::State::Idle:
+			case ChunkserverEntry::State::Read:
+			case ChunkserverEntry::State::GetBlock:
+			case ChunkserverEntry::State::WriteLast:
 				pdesc.emplace_back();
 				pdesc.back().fd = entry.sock;
 				pdesc.back().events = 0;
-				entry.pdescpos = pdesc.size() - 1;
-				if (entry.inputpacket.bytesleft > 0) {
+				entry.pDescPos = pdesc.size() - 1;
+				if (entry.inputPacket.bytesLeft > 0) {
 					pdesc.back().events |= POLLIN;
 				}
-				if (entry.outputhead != NULL) {
+				if (entry.outputHead != nullptr) {
 					pdesc.back().events |= POLLOUT;
 				}
 				break;
-			case CONNECTING:
+			case ChunkserverEntry::State::Connecting:
 				pdesc.emplace_back();
-				pdesc.back().fd = entry.fwdsock;
+				pdesc.back().fd = entry.fwdSocket;
 				pdesc.back().events = POLLOUT;
-				entry.fwdpdescpos = pdesc.size() - 1;
+				entry.fwdPDescPos = pdesc.size() - 1;
 				break;
-			case WRITEINIT:
-				if (entry.fwdbytesleft > 0) {
+			case ChunkserverEntry::State::WriteInit:
+				if (entry.fwdBytesLeft > 0) {
 					pdesc.emplace_back();
-					pdesc.back().fd = entry.fwdsock;
+					pdesc.back().fd = entry.fwdSocket;
 					pdesc.back().events = POLLOUT;
-					entry.fwdpdescpos = pdesc.size() - 1;
+					entry.fwdPDescPos = pdesc.size() - 1;
 				}
 				break;
-			case WRITEFWD:
+			case ChunkserverEntry::State::WriteForward:
 				pdesc.emplace_back();
-				pdesc.back().fd = entry.fwdsock;
+				pdesc.back().fd = entry.fwdSocket;
 				pdesc.back().events = POLLIN;
-				entry.fwdpdescpos = pdesc.size() - 1;
-				if (entry.fwdbytesleft > 0) {
+				entry.fwdPDescPos = pdesc.size() - 1;
+				if (entry.fwdBytesLeft > 0) {
 					pdesc.back().events |= POLLOUT;
 				}
 
 				pdesc.emplace_back();
 				pdesc.back().fd = entry.sock;
 				pdesc.back().events = 0;
-				entry.pdescpos = pdesc.size() - 1;
-				if (entry.inputpacket.bytesleft > 0) {
+				entry.pDescPos = pdesc.size() - 1;
+				if (entry.inputPacket.bytesLeft > 0) {
 					pdesc.back().events |= POLLIN;
 				}
-				if (entry.outputhead != NULL) {
+				if (entry.outputHead != NULL) {
```

Comment:
Here consider changing `NULL` by `nullptr`.

Reply:
outputhead does not exists in next commits. It was replaced by a std::list.